docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -229,6 +229,67 @@ for instance:
     - `OrderStatusDataFactory`
     - `ParameterDataFactory`
     - `UnitDataFactory`
+- (*low priority*) if you want to test flow of coupon code in cart page, implement new acceptance tests ([#825](https://github.com/shopsys/shopsys/pull/825))
+    `Tests\ShopBundle\Acceptance\acceptance\CartCest`
```

Comment:
I am not sure whether it is necessary to provide the diffs here - is not the link to PR sufficient? If you did the fixes based on my comments below, you would need to add more diffs here...

Reply:
more on that, should this be in UPGRADE notes at all? Maybe changelog is sufficient...:thinking:

Reply:
i think that tests for functional discount codes are essential, but who knows.